tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_add_to_an_existing_contact_from_messages.py

```diff
@@ -46,6 +46,7 @@ def test_sms_add_number_to_existing_contact(self):
 
         contacts = Contacts(self.marionette)
         contacts.launch()
+        contacts.wait_for_contacts()
 
```

Comment:
No, I was thinking of when someone tries to access the contacts.contact property, actually. If one would do that, I think we can safely assume that you want at least one contact to be there.